src/meltano/cli/job.py

```diff
@@ -111,6 +120,13 @@ def job(project, ctx):
     """
     ctx.obj["project"] = project
     ctx.obj["task_sets_service"] = TaskSetsService(project)
+    tracker = Tracker(project)
+    tracker.add_contexts(CliContext("job", ctx.invoked_subcommand or None))
+    legacy_tracker = LegacyTracker(project, context_overrides=tracker.contexts)
+    tracker.track_command_event(CliEvent.started)
+    legacy_tracker.track_meltano_job(ctx.invoked_subcommand or None)
```

Comment:
Ah, i think this is the reason we're getting dupe structured events.  Sorry I missed this during the review. We're already firing this same event inline in all the subcommands. 

```suggestion
```